ajax/libs/typeboost.css/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "typeboost.css",
+  "filename": "typeboost.min.css",
```

Comment:
@pvnr0082t Oh yeah that's what I noticed a while ago and was wondering what this property is for. will change